confgenerator/resourcedetector/gce_detector.go

```diff
@@ -128,6 +133,50 @@ func (r GCEResource) MonitoredResource() *monitoredres.MonitoredResource {
 	}
 }
 
+func (r GCEResource) PrometheusStyleMetadata() map[string]string {
+	metaLabels := map[string]string{
+		"__meta_gce_instance_id":   r.InstanceID,
+		"__meta_gce_instance_name": r.InstanceName,
+		"__meta_gce_project":       r.Project,
+		"__meta_gce_zone":          r.Zone,
+		"__meta_gce_network":       r.Network,
+		// TODO(b/b/246995894): Add support for subnetwork label.
```

Comment:
[Optional] Might be a good idea to fix this typo instead of propagating it…
```suggestion
		// TODO(b/246995894): Add support for subnetwork label.
```

Reply:
Actually, never mind, I'll fix that in another PR, and we can pick that up as part of the next merge.